.ci/macos-steps.yaml

```diff
@@ -13,6 +13,8 @@ steps:
 - script: |
     set -e
     sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
+    brew uninstall pkg-config@0.29.2
+    brew install pkg-config
```

Comment:
As this is a hot-fix for a temporary issue, shall we place a 'reminder' somewhere to see about reverting to normal "in due course" ?

Reply:
Good idea, done in aeac96e0d4. :+1: